app/services/person_constructor.rb

```diff
@@ -58,24 +63,21 @@ def flatten_remote_fields
   end
 
   def strip_whitespace_from_values
-    params.merge!(params){ |k, v1| v1.try(:strip) || v1 }
+    params.merge!(params){ |k, v| v.try(:strip) || v }
   end
 
   def normalize_keys
+    mapped_keys = params.keys & KEY_NAME_MAPPINGS.keys
     mapped_keys.each do |key|
       params[ KEY_NAME_MAPPINGS[key] ] = params.delete(key)
     end
   end
 
-  def mapped_keys
-    params.keys & KEY_NAME_MAPPINGS.keys
-  end
-
-  def person_params
+  def person_fields
```

Comment:
I think `person_params` is clearer. I would guess that a method called `person_fields` would return a list of fields on the Person model